### PR TITLE
[MM-14321] Add e2e test for show more/less on long post

### PR DESCRIPTION
**Pull request by thekiiingbob**
#### Summary
* Added e2e test for testing the show more/less functionality on a long post
* Added ID to show more button and the gradient, updated snap shots
* Added 2 fixtures, one the markdown content, the other the html to verify against.

Note, I did not test for a noticeable delay showing or hiding the full post. If we want to do that, I can investigate what would be the best way to go about doing that. Also, would there be a specific time range that is acceptable?

#### Ticket Link
[GitHub Issue](https://github.com/mattermost/mattermost-server/issues/10369)
[JIRA Ticket](https://mattermost.atlassian.net/browse/MM-14321)

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed

### CODE REVIEW

File: cypress/integration/channel/collapsed_message_spec.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+describe('Long message', () => {
+    it('M14321 will show more/less content correctly', () => {
+        // 1. Login as "user-1" and go to /
+        cy.login('user-1');
+        cy.visit('/');
+
+        // 2. Post message with kitchen sink markdown text
+        cy.fixture('kitchenSink.md', 'utf-8').then((text) => {
+            cy.get('#post_textbox').then((textbox) => {
+                textbox.val(text);
```
Comment (thekiiingbob) [inline cypress/integration/channel/collapsed_message_spec.js]: A note on why I entered in the text this way: if I used `type`, it took ~2 minutes to type in the entirety of the markdown file. That felt long to me, so I went with this implementation. By doing it this way I also needed to "trick" the field that it was typed into by doing a space, then backspace, and finally enter to submit.

Comment (saturninoabril) [reply cypress/integration/channel/collapsed_message_spec.js]: Wow, I like this approach.  Fantastic!

File: cypress/fixtures/kitchenSink.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# Basic Markdown Testing
```
Comment (jespino) [inline cypress/fixtures/kitchenSink.md]: Could you rename this fixture file to something more descriptive like `longMarkdownPost.md`?

File: cypress/fixtures/kitchenSink.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
+<h1 class="markdown__heading">Basic Markdown Testing</h1><p>Tests for text style, code blocks, in-line code and images, lines, block quotes, and headings.</p>
```
Comment (jespino) [inline cypress/fixtures/kitchenSink.html]: Could you rename this fixture file to something more descriptive like `longMarkdownPost.html`?

**Comment by thekiiingbob** [comment]
@jespino @saturninoabril Files renamed. Thanks!

**Comment by jespino** [comment]
@thekiiingbob Thanks! great work here! :sparkles:

**Pull request merged**